kubectl/options.go

```diff
@@ -38,30 +54,73 @@ func (options *KubectlOptions) TempConfigFromAuthInfo() (string, error) {
 	}
 	logger.Infof("Created %s to act as temporary kubeconfig file.", tmpfile.Name())
 
+	scheme := options.AuthScheme()
+	switch scheme {
+	case DirectAuth:
+		err = tempConfigFromDirectAuthInfo(
+			logger, tmpfile, options.Server, options.Base64PEMCertificateAuthority, options.BearerToken)
+	case EKSClusterBased:
+		err = tempConfigFromEKSClusterInfo(logger, tmpfile, options.EKSClusterArn)
+	default:
+		return "", errors.WithStackTrace(AuthSchemeNotSupported{scheme})
+	}
+	return tmpfile.Name(), err
+
```

Comment:
```suggestion

	return tmpfile.Name(), err
```

Reply:
In case it's unclear, the suggestion here is just to add a new line after the closed `switch` and make the `return` the last line of the function.